src/WormholeJoin.sol

```diff
@@ -224,6 +244,35 @@ contract WormholeJoin {
             vat.slip(ilk, address(this), -int256(amtToPayBack));
         }
         debt[sourceDomain] -= int256(batchedDaiToFlush);
+        rakes[sourceDomain] += batchedDaiToFlush;
         emit Settle(sourceDomain, batchedDaiToFlush);
     }
+
+    /**
+    * @dev Rectify bad debt by pulling dai from the vow.
+    * @param sourceDomain domain to rectify
+    * @param era the era in which the bad debt occurred.
+    **/
+    function rectify(bytes32 sourceDomain, uint256 era) external {
+        require(vat.live() == 1, "WormholeJoin/vat-not-live");
+        require(era * bucket[sourceDomain] + ttl[sourceDomain] <= block.timestamp, "WormholeJoin/ttl-not-expired");
+
+        uint256 eraDebt = eras[sourceDomain][era];
+        uint256 daiSettled = rakes[sourceDomain];
+        require(daiSettled < eraDebt, "WormholeJoin/debt-repaid");
+
+        uint256 badDebt = eraDebt - daiSettled;
+        require(badDebt <= 2 ** 255, "WormholeJoin/overflow");
+        rakes[sourceDomain] = daiSettled + badDebt;
+        vat.suck(vow, address(this), badDebt * RAY);
```

Comment:
I think we should suck just whatever is necessary for covering the debt, so calculate `min(badDebt, art)` before.

Reply:
What's the issue with just adding the excess? I need to think about this more, but I don't think we should be artificially capping the bad debt reconciliation.